### ISSUE TITLE
Import clients with "Authorization Enabled" (fine-grained authorization support)

### ISSUE THREAD

**Issue by rpiotrow** [adorsys/keycloak-config-cli#144]
**Is your feature request related to a problem? Please describe.**

It is not possible to import realm when some client in it has Authorization Enabled switch "On" (means support for fine-grained authorization support for that client)

**Comment by jkroepke** [comment]
Hi

Whats the error?

Do you have an import example?

**Comment by rpiotrow** [comment]
Error is quite generic. I'll try to prepare some example soon.

**Comment by jkroepke** [comment]
Using an example from https://raw.githubusercontent.com/keycloak/keycloak-quickstarts/latest/app-authz-jee-servlet/config/realm-import.json works without an error.

**Comment by jkroepke** [comment]
Okay,

while write some test with "Authorization Enabled" clients, I got some errors.

**Comment by jkroepke** [comment]
@rpiotrow 

I implement some features which coders the fine-grained authorization settings on a client.

The code is on master. You could test it with the docker image `adorsys/keycloak-config-cli:master` or downloading the artefact from here: https://github.com/adorsys/keycloak-config-cli/actions/runs/183241781

**Comment by rpiotrow** [comment]
Still getting error using `adorsys/keycloak-config-cli:master`. Steps to reproduce:
* create fresh test realm
* create test client (standard flow disabled, type confidential, fine-grained authorization enabled)
* export test realm into json file
* try to import above file (also attached) into fresh (empty) keyclaok instance

Error:
```
Cannot create realm 'test': Unrecognized field "clientOfflineSessionIdleTimeout" (class org.keycloak.representations.idm.RealmRepresentation), not marked as ignorable
```

Keycloak version: `jboss/keycloak:10.0.2`

**Comment by jkroepke** [comment]
Hi,

> Cannot create realm 'test': Unrecognized field "clientOfflineSessionIdleTimeout" (class org.keycloak.representations.idm.RealmRepresentation), not marked as ignorable

The `master` tag is also build with the latest Keycloak version (11.0.0). Older keycloak version are only available as jar files or in releases.

But i can confirm running master (builded with keycloak 10.0.2) runs successfully with your test-realm. 

What I do:
```
git clone https://github.com/adorsys/keycloak-config-cli.git
cd keycloak-config-cli
mvn clean package -Dkeycloak.version=10.0.2
env KEYCLOAK_VERSION=10.0.2 docker-compose up -d keycloak # start a local keycloak on docker
java -jar target/keycloak-config-cli.jar --keycloak.url=http://localhost:8080 --keycloak.password=admin123 --import.path=test-realm.json
```

Log output:
```
2020-07-31 12:28:00.987  INFO 93669 --- [           main] d.a.k.config.KeycloakConfigApplication   : Starting KeycloakConfigApplication v2.1.1-SNAPSHOT on joe-nb with PID 93669 (/Users/jkr/Documents/keycloak-config-cli/target/keycloak-config-cli.jar started by jkr in /Users/jkr/Documents/keycloak-config-cli)
2020-07-31 12:28:00.991  INFO 93669 --- [           main] d.a.k.config.KeycloakConfigApplication   : No active profile set, falling back to default profiles: default
2020-07-31 12:28:01.550  INFO 93669 --- [           main] d.a.k.config.KeycloakConfigApplication   : Started KeycloakConfigApplication in 0.988 seconds (JVM running for 1.343)
2020-07-31 12:28:01.831  INFO 93669 --- [           main] d.a.k.c.provider.KeycloakImportProvider  : Importing file '/Users/jkr/Documents/keycloak-config-cli/test-realm.json'
2020-07-31 12:28:06.240  INFO 93669 --- [           main] d.a.k.config.KeycloakConfigRunner        : keycloak-config-cli running in 00:04.429.
```

On Keycloak I could found a test realm and a test client.

**Comment by rpiotrow** [comment]
I've tried docker image `adorsys/keycloak-config-cli:latest-10.0.2` but that gived me:
```
2020-07-31 10:56:36.288  INFO 1 --- [           main] d.a.k.config.KeycloakConfigApplication   : Starting KeycloakConfigApplication v2.1.0 on rpiotrow with PID 1 (/opt/keycloak-config-cli.jar started by ? in /)
2020-07-31 10:56:36.304  INFO 1 --- [           main] d.a.k.config.KeycloakConfigApplication   : No active profile set, falling back to default profiles: default
2020-07-31 10:56:38.117  INFO 1 --- [           main] d.a.k.config.KeycloakConfigApplication   : Started KeycloakConfigApplication in 3.776 seconds (JVM running for 4.836)
2020-07-31 10:56:39.271  INFO 1 --- [           main] d.a.k.c.provider.KeycloakImportProvider  : Importing file '/config/test-realm.json'
2020-07-31 10:56:39.705  INFO 1 --- [           main] d.a.k.config.provider.KeycloakProvider   : Wait 120 seconds until http://localhost:9080 is available ...
2020-07-31 10:56:43.238 ERROR 1 --- [           main] d.a.k.config.KeycloakConfigRunner        : Cannot create realm 'test': {"error":"unknown_error"}
2020-07-31 10:56:43.239  INFO 1 --- [           main] d.a.k.config.KeycloakConfigRunner        : keycloak-config-cli running in 00:04.049.
```
I'll try later to clone, build and run locally (as you do) to compare it.

**Comment by jkroepke** [comment]
Sure, because the changes from https://github.com/adorsys/keycloak-config-cli/pull/149 are not released yet and not included into "latest".

**Comment by jkroepke** [comment]
This feature is released as 2.2.0 now.
